server/engine/src/engine.py

```diff
@@ -52,43 +57,56 @@ def main():
         # segments that have not yet thrown an error during loading.
         print('Running kinematics fit', flush=True)
         subject.run_kinematics_fit(DATA_FOLDER_PATH)
```

Comment:
If we use the term `run_dynamics_pass` (see comment below), consider updating this to `run_kinematics_pass` for uniformity.

Reply:
Done